android/app/src/main/java/io/peggyjo/peggymeter/FirstTimeActivity.java

```diff
@@ -30,7 +33,21 @@ protected void onCreate(Bundle savedInstanceState) {
             controller.getSettingAdapter().setProperty(OPT_IN_PROPERTY, false);
             finish();
         });
+
         ((WebView) findViewById(R.id.webView)).loadUrl("file:///android_asset/welcome.html");
+
+        // Adds a button that will crash the app
```

Comment:
Can you put this code in
  if( BuildConfig.DEBUG ){
  }

so it runs in debug mode only?